client/src/document/ingredients/browser-compatibility-table/atoms/_bc-supports.scss

```
@@ -84,3 +84,7 @@ $bc-partial-support-cross: rgba(144, 99, 18, 0.2);
 .bc-supports-unknown {
   background-color: $neutral-550;
 }
+
+.bc-supports-preview {
+  background-color: $primary-500;
```

This color doesn't exist any longer in the latest `@mdn/minimalist`. I think the closest equivalent in the latest color palette is `$mdn-blue100`? (cc @schalkneethling)
```suggestion
  background-color: $mdn-blue100;
```